src/Components/TicTacToe/Board.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
No need to import ReactDom. It is only needed in the index.js component where App.js which is the root component of our app gets rendered to the actual DOM. It's not required for any of the child components.